extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc

```diff
@@ -18,6 +18,7 @@ Contributors
 - Hai Nguyen, Google
 - David Neto, Google
 - Ehsan Nasiri, Google
+- Natalie Chouinard, Google
 
 Status
 ------
```

Comment:
Version table below this needs to be updated.

Reply:
Do you mean the Revision History table at the bottom of this file? I added a row to it.

Reply:
No, I mean lines 32 and 33, which are collapsed in the diff and I can't comment on them. 
They should be updated to match the latest entry in the Revision History table.

Reply:
Oops, don't know how I missed that on the second look too. Thanks!